src/sqlittaaja/config.py

```diff
@@ -8,7 +8,7 @@ class Config:
     """Application level configuration options."""
 
     initialize_script: str = ""
-    exercises: list[(str, str)] = []
+    exercises: list[(str, str, list[str])] = []
```

Comment:
Isn't this suppose to be `list[(str, str, list[str], list[str])]` like down below?